### PR TITLE
Move http docs to README

### PR DESCRIPTION
**Pull request by madhuravi**

### CODE REVIEW

**Comment by madhuravi** [comment]
@anuptalwalkar

File: modules/uhttp/doc.go [hunk 1]
```diff
@@ -48,19 +47,19 @@
 //   }
 //
 //   func registerHTTP(service service.Host) []uhttp.RouteHandler {
-//     handleHome := http.HandlerFunc(func(ctx fx.Context, w http.ResponseWriter, r *http.Request) {
```
Comment (anuptalwalkar) [inline modules/uhttp/doc.go]: you missed these

File: modules/uhttp/README.md [hunk 2]
```diff
@@ -35,4 +35,9 @@ func registerHTTP(service service.Host) []uhttp.RouteHandler {
     uhttp.NewRouteHandler("/", handleHome)
   }
 }
+HTTP handlers are set up with filter chains to inject tracing, authentication information etc.
+into the request context. This is abstracted away from the client. This also sets up the request
+with a context-aware logger. So all service log statements include trace information such as
+traceID and spanID. This helps service owners with debugging a request and further corelating
```
Comment (alsamylkin) [inline modules/uhttp/README.md]: typo, correlating

Comment (alsamylkin) [inline modules/uhttp/README.md]: Also can we rephrase the message a little more, too many sentences start with This :)

File: modules/uhttp/README.md [hunk 3]
```diff
@@ -27,12 +28,18 @@ func main() {
...
+    ctx.Logger().Info("Inside the handler")
     io.WriteString(w, "Hello, world")
   })
 
   return []uhttp.RouteHandler{
     uhttp.NewRouteHandler("/", handleHome)
   }
 }
+
+HTTP handlers are set up with filters that inject tracing, authentication information etc. into the
```
Comment (sectioneight) [inline modules/uhttp/README.md]: This belongs outside the cod block (below line 45). See the readme as merged: https://github.com/uber-go/fx/tree/master/modules/uhttp

**Pull request merged**